sockeye/arguments.py

```diff
@@ -1063,6 +1063,11 @@ def add_training_args(params):
                               default=-1,
                               help='Keep only the last n params files, use -1 to keep all files. Default: %(default)s')
 
+    train_params.add_argument('--keep-initializations',
+                              type=bool,
```

Comment:
could you use `action='store_true'` to be consistent with other boolean cmdline flags?